src/indexed_crate.rs

```diff
@@ -41,6 +41,218 @@ pub struct IndexedCrate<'a> {
     pub(crate) manually_inlined_builtin_traits: HashMap<Id, Item>,
 }
 
+#[cfg(feature = "rayon")]
+use rayon::prelude::*;
```

Comment:
nit: it would be a bit better to move this adjacent to all the other imports, above `struct IndexedCrate`

Reply:
Fixed!